ert/data/record/_record.py

```diff
@@ -59,19 +59,51 @@ class RecordType(str, Enum):
 
 
 class Record(ABC):
+    """The :py:class:`Record` class is an abstract class that
+    includes record data and the record type.
+
+    The record needs be either `BlobRecord` or `NumericalRecord` type derived
+    from `Record` class
+    """
+
     @property
     @abstractmethod
     def data(self) -> record_data:
+        """Returns record data.
+
+        Returns:
+            record_data: record data as
+            Union[numerical_record_data, blob_record_data]
+        """
         pass
 
     @property
     @abstractmethod
     def record_type(self) -> RecordType:
+        """Returns the type of the record data.
+
+        Returns:
+            RecordType: type of data
+        """
         pass
 
 
 class BlobRecord(Record):
+    """The :py:class:`BlobRecord` class is realization of the Record class
```

Comment:
We should probably not use the term `realization` for this in ERT.. Instantiation or something?

Reply:
Concrete implementation of…?

Reply:
Right, `realization` might be a confusing term in the case will update to `implementation` instead.

Reply:
Since concrete is an antonym of abstract (and this is called a concrete subclass of the abstract superclass) it would be nice to include the full suggestion of @jondequinor, but of course it is not vital and already good as it is.